exercises/concept/language-list/LanguageListTest.php

```diff
@@ -118,6 +118,17 @@ public function testGettingFirstDoesNotMutate()
      * @task_id 6
      */
     public function testLanguageListCount()
+    {
+        $language_list = language_list('c', 'cpp', 'php');
+        $languages_count = language_list_length($language_list);
+        $this->assertEquals(3, $languages_count);
+    }
+
+    /**
+     * @testdox when getting the language count, original is unchanged
+     * @task_id 7
```

Comment:
This is part of the 6th task (see `.docs/instructions.md`). There is no 7, yet :smiley: 

```suggestion
     * @task_id 6
```